test/utils/snap.go

```diff
@@ -69,6 +69,14 @@ func SnapVersion(t *testing.T, name string) string {
 	return strings.TrimSpace(out)
 }
 
+func SnapRevision(t *testing.T, service string) string {
+	out, _ := exec(t, fmt.Sprintf(
+		`snap run --shell %s -c "echo \$SNAP_REVISION"`,
```

Comment:
This is very hacky. How about using snap info (like we did for getting the version) or snap list?
```
$ snap info edgex-device-usb-camera | grep installed | awk '{print $3}'
(55)
$ snap list | grep edgex-device-usb-camera | awk '{print $3}'
55
```

Using `snap info` is more reliable because it won't match something else by chance and returns error.